app/cdap/components/ConnectionList/index.tsx

```diff
@@ -68,13 +73,20 @@ export default function ConnectionList() {
       isSearching: false,
     },
   ]);
+  const [filteredData, setFilteredData] = useState(cloneDeep(dataForTabs));
 
   const getConnectionsTabData = async () => {
-    // Fetching the all available connectors list
-    let connectorTypes = await fetchConnectors();
+    let connectorTypes = [];
+    let connectorTypesWithSVG = [];
```

Comment:
add types for variables if it is not related to API

Reply:
need to check